sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Linq/DigitalTwinsQueryBuilder.cs

```diff
@@ -0,0 +1,406 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Globalization;
+using System.Text;
+using Azure.Core;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder.Linq
+{
+    /// <summary>
+    /// Azure DigitalTwins Query builder that facilitates writing queries against ADT instances. Redirects to generic version of this class using BasicDigitalTwin as type T.
+    /// </summary>
+    public class DigitalTwinsQueryBuilder : DigitalTwinsQueryBuilder<BasicDigitalTwin>
+    {
+        /// <summary>
+        /// Create a DigitalTwins query and set the queried collection to DigitalTwins by default.
+        /// </summary>
+        public DigitalTwinsQueryBuilder() : base(DigitalTwinsCollection.DigitalTwins) { }
+
+        /// <summary>
+        /// Create a DigitalTwins query and set the queried collection to a custom string.
+        /// </summary>
+        /// <param name="customColection">Collection to query from.</param>
+        public DigitalTwinsQueryBuilder(string customColection) : base(customColection) { }
+
+        /// <summary>
+        /// Create a DigitalTwins query and set the queried collection to DigitalTwins or Relationships.
+        /// </summary>
+        /// <param name="collection">Collection to query from.</param>
+        /// <param name="alias">Optional collection alias.</param>
+        public DigitalTwinsQueryBuilder(DigitalTwinsCollection collection, string alias = null) : base(collection, alias) { }
+    }
+
+    /// <summary>
+    /// Azure DigitalTwins Query builder that facilitates writing queries against ADT instances.
+    /// </summary>
+    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:File may only contain a single type", Justification = "Required for default behavior")]
```

Comment:
Nit - we usually use the `#pragma` suppressions for these because you can bump them to the top of the file.